tools/WebStack.versions.settings.targets

```diff
@@ -15,7 +15,7 @@
     <AspNetCorePackageDependency>[2.0.0, 3.0.0)</AspNetCorePackageDependency>
     <DependencyInjection1PackageDependency>1.0.0</DependencyInjection1PackageDependency>
     <DependencyInjection2PackageDependency>2.0.0</DependencyInjection2PackageDependency>
-    <ODataLibPackageDependency>[7.9.0, 8.0.0)</ODataLibPackageDependency>
+    <ODataLibPackageDependency>[7.10.0, 8.0.0)</ODataLibPackageDependency>
```

Comment:
what's the dependency in our csproj? 
You should make sure the version is same between them.